site/en/setup.md

```diff
@@ -169,6 +171,9 @@ regularly update the repository and its submodules:
 git pull
 ```
 
+This will download or update all submodules and provide a list specifying the amount of changes in all submodules.
```

Comment:
`git pull` will not update all submodules. Did you think of `make modules` rather than `git pull` perhaps?

Reply:
oops, you are right. But then I think we need to change the documentation even more. `make all` ist only listed unter the installation paragraph. So without prior knowledge I would assume that it is only needed for the installation once.

Reply:
It is. `make all` implies `make modules`, so the git submodules will be initialized and updated automatically.

`git pull` does update the references to the git submodules, so git "knows" that the local submodules need updating but the actual updates require calls to `git submodule sync` (in case the URL of the repo changed) and `git submodule update --init` - that's exactly what `make modules` and by proxy `make all` does.

Maybe:

> This will refresh the local clone of `ocrd_all` with the changes in the official ocrd_all GitHub repository.